docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/ecommerce-tracking/index.md

```diff
@@ -0,0 +1,702 @@
+| totalQuantity  | Y        | integer     | Number of items in the transaction.        |
+| tax            | N        | number      | Tax value in transaction.                  |
+| shipping       | N        | number      | Shipping costs in transaction.             |
+| discountCode   | N        | string      | An applied discount code.                  |
+| discountAmount | N        | string      | Amount discounted from transaction.        |
+| creditOrder    | N        | boolean     | Whether the transaction is a credit order. |
+
+</details>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.ecommerce/transaction/jsonschema/1-0-0`.
+
+### Refund entity
+
+The refund entity is used for `Refund` events.
+
+<details>
+    <summary>Refund entity properties</summary>
+
+| Request Key   | Required | Type/Format | Description                                   |
+|---------------|----------|-------------|-----------------------------------------------|
+| transactionId | Y        | string      | The identifier from the original transaction. |
+| refundAmount  | Y        | number      | Amount to be refunded.                        |
+| currency      | Y        | string      | Currency used in transaction.                 |
+| refundReason  | N        | string      | Reason for the refund.                        |
+
+</details>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.ecommerce/refund/jsonschema/1-0-0`.
+
+### Promotion entity
+
+The promotion entity is used for `PromotionView` and `PromotionClick` events.
+
+<details>
+    <summary>Promotion entity properties</summary>
+
+| Request Key | Required | Type/Format     | Description                                        |
+|-------------|----------|-----------------|----------------------------------------------------|
+| id          | Y        | string          | The unique promotion identifier.                   |
+| name        | N        | string          | Promotion name.                                    |
+| productIds  | N        | list of strings | IDs of products in the promotion.                  |
+| position    | N        | integer         | Index of the promotion in a list such as a banner. |
+| creativeId  | N        | string          | Identifier for the promotion creative.             |
+| type        | N        | string          | Type of promotion.                                 |
+| slot        | N        | string          | Where the promotion was displayed.                 |
+
+</details>
+
+*Schema:*
+`iglu:com.snowplowanalytics.snowplow.ecommerce/promotion/jsonschema/1-0-0`.
+
+## Global ecommerce entities Screen/Page and User
+
+Use these APIs to add ecommerce context information to every subsequent event tracked.
+
+:::note
+These entities will be added to **all** events, not just ecommerce ones.
+:::
+
+### EcommerceUser entity
+
+To set an Ecommerce User entity you can use the `setEcommerceUser` method with the following attributes:
```

Comment:
Nit: I don't know if we can give this intuition here, but if so, can we explain what is the purpose of this entity as opposed to setting the `userId` in Subject or doing it some other way? Is there a particular reason that the user should be described using this entity in ecommerce that would help users know when to use it?